lib/galaxy/jobs/runners/pulsar.py

```diff
@@ -63,6 +64,10 @@
         valid=specs.is_in("urllib", "curl", None),
         default=None
     ),
+    transport_timeout=dict(
+        map=lambda val: None if val == "None" else int(val),
```

Comment:
Let's accept as is and I'll submit a second PR once it's merged to dev and fix it there?